components/authentication/base/rhtap-admins.yaml

```diff
@@ -0,0 +1,215 @@
+      - imagestreammappings
+      - imagestreams
+      - imagetags
+      - limitranges
+      - persistentvolumeclaims
+      - pods
+      - pods/attach
+      - pods/exec
+      - pods/port-forward
+      - processedtemplates
+      - replicationcontrollers
+      - replicationcontrollers/scale
+      - resourcequotas
+      - rolebindings
+      - roles
+      - routes
+      - serviceaccounts
+      - services
+      - templateconfigs
+      - templateinstances
+      - templates
+    verbs:
+      - '*'
+  - apiGroups:
+      - 'apps'
+    resources:
+      - daemonsets
+      - deployments
+      - deployments/scale
+      - replicasets
+      - replicasets/scale
+      - statefulsets
+      - statefulsets/scale
+    verbs:
+      - '*'
+  - apiGroups:
+      - 'batch'
+    resources:
+      - cronjobs
+      - jobs
+    verbs:
+      - '*'
+  - apiGroups:
+      - extensions
+    resources:
+      - daemonsets
+      - deployments
+      - deployements/rollback
+      - deployments/scale
+      - ingresses
+      - networkpolicies
+      - replicasets
+      - replicasets/scale
+      - replicationcontrollers/scale
+    verbs:
+      - '*'
+  - apiGroups:
+      - apps.openshift.io
+    resources:
+      - deploymentconfigs
+      - deploymentconfigs/scale
+    verbs:
+      - '*'
+  - apiGroups:
+      - authorization.openshift.io
+    resources:
+      - roles
+      - rolebindings
+    verbs:
+      - '*'
+  - apiGroups:
+      - build.openshift.io
+    resources:
+      - builds
+      - buildconfigs
+      - buildconfigs/webhooks
+      - buildlogs
+    verbs:
+      - '*'
+  - apiGroups:
+      - image.openshift.io
+    resources:
+      - imagestreams
+      - imagestreamimages
+      - imagestreammappings
+      - imagetags
+    verbs:
+      - '*'
+  - apiGroups:
+      - route.openshift.io
+    resources:
+      - routes
+    verbs:
+      - '*'
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+      - rolebindings
+      - roles
+    verbs:
+      - '*'
+  - apiGroups:
+      - autoscaling
+    resources:
+      - horizontalpodautoscalers
+    verbs:
+      - '*'
+  - apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - alertmanagers
+      - prometheuses
+      - prometheusrules
+      - servicemonitors
+      - podmonitors
+      - thanosrulers
+    verbs:
+      - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: rhtap-admins
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rhtap-admins
+subjects:
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: amfred
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: bamachrn
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: gbenhaim
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: hugares
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: manish-jangra
```

Comment:
should we list infra team one by one or we should grant access to "Infra Team" github group instead?

Reply:
I think we'll need to factor out individuals' Github user IDs over time into another repo, and a good first step along that path would be to grant access to a group instead of individuals.

Reply:
Done, as per other comment